docs/self-hosted-getting-started.md

```diff
@@ -0,0 +1,99 @@
+# Quick Start: Run a self-hosted database instance with automatic vectorization
+
+If you want to see how it works, follow these steps to setup 
+a self-hosted database instance with automatic embedding vectorization:
+
+## Pre-requisites
+- Docker installed on your machine
+- An OpenAI API key. You can create one [here](https://platform.openai.com/api-keys).
+
+## Setup
+
+1. Run the following docker-compose file to start a TimescaleDB instance and the Vectorizer worker: 
+    ```yaml
+    services:
```

Comment:
I'd add `name: pgai` so that we can assume `pgai-` will be the prefix for all the containers.